salt/transport/tcp.py

```diff
@@ -531,10 +532,10 @@ def _create_stream(
         # after one connection has completed.
         sock = _get_socket(self.opts)
         _set_tcp_keepalive(sock, self.opts)
-        stream = salt.ext.tornado.iostream.IOStream(
+        stream = tornado.iostream.IOStream(
             sock, max_buffer_size=max_buffer_size
         )
-        if salt.ext.tornado.version_info < (5,):
+        if tornado.version_info < (5,):
```

Comment:
We are now forcing >=6.1 right?
Remove check?

Reply:
Good catch, thanks